### PR TITLE
Fixes #1867. TextView: Fixes wordwrap on LoadFile and fixes extra ending space when deleting backwards.

### LINKED ISSUE #1867: Use Undo and Redo commands with WordWrap enabled
**Is your feature request related to a problem? Please describe.**
No, it's just a feature, which in this case would be to be able to use Undo and Redo commands with WordWrap activated in a TextView.

### PR DESCRIPTION
**Pull request by BDisp**
Fixes #1867.

### CODE REVIEW

File: UnitTests/TextViewTests.cs [hunk 1]
```diff
@@ -1962,19 +1968,74 @@ public void WordWrap_WrapModel_Output ()
...
-This is 
-the 
-first 
-line.
-This is 
-the 
-second 
+This is
+the first
+ line.
```
Comment (tig) [inline UnitTests/TextViewTests.cs:1973]: Per #1820, we all agree that TextView is not RENDERING word-wrapped text correctly. The correct rendering is in the deleted text here where the space was removed from the ` line.` line.

Right?

So, once the RENDERING behavior is fixed, this unit test will need to change.

I'd like you to rename these unit test methods to be clear on what is actually being tested.

In this case, the unit test is testing RENDERING.

There are (should be?) other unit tests that test that the INTERNAL REPRESENTATION of the `Text` is correct.

**Review by tig — CHANGES_REQUESTED** [review]

File: UnitTests/TextViewTests.cs (hunk 1 above)
Comment (BDisp) [reply UnitTests/TextViewTests.cs:1973]: The space is removed after set ReadOnly to true. But I'm seeing you didn't push all the code changes that I submitted to your pr branch. I'll submit a new pr to the gui-cs.

**Comment by BDisp** [comment]
Merging one of them will also merging the other.

File: UnitTests/TextViewTests.cs (hunk 1 above)
Comment (tig) [reply UnitTests/TextViewTests.cs:1973]: > The space is removed after set ReadOnly to true. But I'm seeing you didn't push all the code changes that I submitted to your pr branch. I'll submit a new pr to the gui-cs.

You are not understanding me. 

The `ReadOnly` fix is a temporary HACK that will fix the way TextView *RENDERS* when WordWrap is enabled.

This DOES NOT FINALLY FIX #1820.  #1820 will ONLY be fixed correctly once we do a massive change to TextView that separates RENDERING of `Text` from how `Text` is stored internally. 

In THIS PR (#1867) I am requesting that you change the unit test NAMES to make it clear that what they are testing is just the RENDERING.

**Review by tig — CHANGES_REQUESTED** [review]

File: UnitTests/TextViewTests.cs (hunk 1 above)
Comment (BDisp) [reply UnitTests/TextViewTests.cs:1973]: @tig I hope have done what you asked. I'm a bit confused :-)

**Comment by BDisp** [comment]
@tig, is there any problem with this pr?

**Pull request closed without merging**